docs/docs/09.1-language-tooling.md

```diff
@@ -0,0 +1,81 @@
+---
+id: language-tooling
+title: Language Tooling
+permalink: language-tooling.html
+prev: tooling-integration.html
+next: package-management.html
+---
+
+## ES6 with JSX
+
+### In-browser JSX Transform
+
+If you like using JSX, Babel 5 provided an in-browser ES6 and JSX transformer for development called browser.js that can be included from [CDNJS](http://cdnjs.com/libraries/babel-core/5.8.34). Include a `<script type="text/babel">` tag to engage the JSX transformer.
+
+> Note:
+>
+> The in-browser JSX transformer is fairly large and results in extraneous computation client-side that can be avoided. Do not use it in production — see the next section.
+
+There is also an unofficial/unsupported [Babel6 in-browser transformer](http://www.jimsproch.com/react/), which includes the JSX presets and also enables some of the newer features like `__source` for better debug messages.
```

Comment:
I don't want to reference this even if it is labeled unofficial and unsupported. If it's in our docs that gives it weight. I especially don't want jsx-control-statements to be in something we link to. Let's instead build a real babel6 transformer that doesn't have any opinions in it (apart from supporting the basics needed for JSX & es2015, maybe __source)